src/main/java/com/assure/movie/service/MovieRentalService.java

```diff
@@ -2,61 +2,89 @@
 
 import com.assure.movie.common.enums.StatusRental;
 import com.assure.movie.common.errors.NotFoundErrorException;
-import com.assure.movie.model.domain.Member;
-import com.assure.movie.model.domain.Movie;
 import com.assure.movie.model.domain.MovieCatalog;
 import com.assure.movie.model.domain.MovieRental;
 import com.assure.movie.model.repository.MovieRentalRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
 /**
  * @author Israel Yasis
  */
 @Service
 public class MovieRentalService {
 
+    private final static int DAYS_TO_RETURN = 3;
+
     private MovieRentalRepository movieRentalRepository;
     private MovieService movieService;
     private MemberService memberService;
+    private MovieCatalogService movieCatalogService;
 
     @Autowired
     public MovieRentalService(
         MovieRentalRepository movieRentalRepository,
         MovieService movieService,
-        MemberService memberService
+        MemberService memberService,
+        MovieCatalogService movieCatalogService
     ) {
         this.memberService = memberService;
         this.movieService = movieService;
         this.movieRentalRepository = movieRentalRepository;
+        this.movieCatalogService = movieCatalogService;
     }
     private MovieRental getMovieRental(Long id) throws NotFoundErrorException {
         return this.movieRentalRepository.findById(id).orElseThrow( ()
                 -> new NotFoundErrorException("Not found movie rental with id: "+ id));
     }
 
+    public List<MovieRental> getMovieRentals() {
+        return this.movieRentalRepository.findAll();
+    }
+
     public MovieRental returnMovieRental(Long id, StatusRental statusRental) {
```

Comment:
You are not using the _statusRental_ parameter, I understood that the status will be provided by the REST endpoint in order the API update the data with that value.

Reply:
yeap, you are right I must receive the status from the controller and the pass it to the service